torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -245,10 +247,12 @@ struct ChunkDatasetOptions {
   ChunkDatasetOptions(
       size_t preloader_count,
       size_t batch_size,
-      size_t cache_size = 2048)
+      size_t cache_size = 2048,
+      std::string resume_from_file = "")
```

Comment:
I'll add a comment in the code explaining what this 'resume_from_file' variable do